src/bun.js/bindings/ErrorCode.cpp

```diff
@@ -420,7 +420,7 @@ JSC::EncodedJSValue INVALID_ARG_TYPE(JSC::ThrowScope& throwScope, JSC::JSGlobalO
 {
     auto arg_name = val_arg_name.toWTFString(globalObject);
     RETURN_IF_EXCEPTION(throwScope, {});
-    auto arg_kind = arg_name.startsWith("options."_s) ? "property"_s : "argument"_s;
+    auto arg_kind = arg_name.contains("."_s) ? "property"_s : "argument"_s;
```

Comment:
Use `'.'` instead of `""_s` since it's a single byte
```suggestion
    auto arg_kind = arg_name.contains('.') ? "property"_s : "argument"_s;
```